config/surveys/lsst_dc2.yaml

```diff
@@ -2,14 +2,36 @@
 name: lsst
 release: dc2
 survey_files: 
+    # Path to data files (leave empty for default location)
     file_path: ''
+
+    # Band-specific magnitude limit maps
     maglim_map_g: supreme_dc2_dr6d_v3_g_maglim_psf_wmean.fits.gz
     maglim_map_r: supreme_dc2_dr6d_v3_r_maglim_psf_wmean.fits.gz
```

Comment:
Are we providing these files in the repo? If not we should at least mention where they can be found in desc land